### PR TITLE
Add visual monitor.

### PR DESCRIPTION
**Pull request by mldk-tech**
#1

### CODE REVIEW

**Comment by mldk-tech** [comment]
@amitaibu - ^^

File: visual-monitor/test/tests.js [hunk 1]
```diff
@@ -12,13 +12,6 @@ var capsConfig = {
     'os' : 'OS X',
     'os_version' : 'Yosemite',
     'resolution' : '1024x768'
-  },
-  'ie11': {
```
Comment (amitaibu) [inline visual-monitor/test/tests.js:16]: please keep

File: visual-monitor/test/tests.js [hunk 2]
```diff
@@ -46,38 +39,42 @@ describe('Visual monitor testing', function() {
   it('should show the home page',function(done) {
     client
       .url(baseUrl)
+      .pause(3000)
+      .click('#_ZAhelloBar #_ZAhelloBarRightImg')
```
Comment (amitaibu) [inline visual-monitor/test/tests.js:51]: add comment

**Comment by mldk-tech** [comment]
@amitaibu - ^^

**Pull request merged**